File: module5/bai_thi/spring/src/main/resources/webapp/angular-final-test/src/app/model/project.ts
export class Project {
  private _id: number;
  private _nameStudent: string;
  private _nameGroup: string;
  private _nameProject: string;
  private _nameTeacher: string;
  private _email: string;
  private _phone: string;


  constructor(id: number, nameStudent: string, nameGroup: string, nameProject: string, nameTeacher: string, email: string, phone: string) {
    this._id = id;
    this._nameStudent = nameStudent;
    this._nameGroup = nameGroup;
    this._nameProject = nameProject;
    this._nameTeacher = nameTeacher;
    this._email = email;
    this._phone = phone;
  }


  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get nameStudent(): string {
    return this._nameStudent;
  }

  set nameStudent(value: string) {
    this._nameStudent = value;
  }

  get nameGroup(): string {
    return this._nameGroup;
  }

  set nameGroup(value: string) {
    this._nameGroup = value;
  }

  get nameProject(): string {
    return this._nameProject;
  }

  set nameProject(value: string) {
    this._nameProject = value;
  }

  get nameTeacher(): string {
    return this._nameTeacher;
  }

  set nameTeacher(value: string) {
    this._nameTeacher = value;
  }

  get email(): string {
    return this._email;
  }

  set email(value: string) {
    this._email = value;
  }

  get phone(): string {
    return this._phone;
  }

  set phone(value: string) {
    this._phone = value;
  }
}

File: module5/bai_thi/spring/src/main/java/com/example/demo/model/Project.java
package com.example.demo.model;

import javax.persistence.*;

@Entity
@Table(name = "project")
public class Project {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    private String nameStudent;
    private String nameGroup;
    private String nameProject;
    private String nameTeacher;
    private String email;
    private String phone;

    public Project(Integer id, String nameStudent, String nameGroup, String nameProject, String nameTeacher, String email, String phone) {
        this.id = id;
        this.nameStudent = nameStudent;
        this.nameGroup = nameGroup;
        this.nameProject = nameProject;
        this.nameTeacher = nameTeacher;
        this.email = email;
        this.phone = phone;
    }

    public Project(String nameStudent, String nameGroup, String nameProject, String nameTeacher, String email, String phone) {
        this.nameStudent = nameStudent;
        this.nameGroup = nameGroup;
        this.nameProject = nameProject;
        this.nameTeacher = nameTeacher;
        this.email = email;
        this.phone = phone;
    }

    public Project() {
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getNameStudent() {
        return nameStudent;
    }

    public void setNameStudent(String nameStudent) {
        this.nameStudent = nameStudent;
    }

    public String getNameGroup() {
        return nameGroup;
    }

    public void setNameGroup(String nameGroup) {
        this.nameGroup = nameGroup;
    }

    public String getNameProject() {
        return nameProject;
    }

    public void setNameProject(String nameProject) {
        this.nameProject = nameProject;
    }

    public String getNameTeacher() {
        return nameTeacher;
    }

    public void setNameTeacher(String nameTeacher) {
        this.nameTeacher = nameTeacher;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }
}
